src/web/screens/sharingPermissions.tsx

```diff
@@ -99,11 +99,15 @@ function SharingPermissions() {
                   />
                 </Collapsible>
               </div>
-              <SharingPermissionsTable
-                sharingParticipants={sharingParticipants}
-                onDeleteSharingPermission={handleDeleteSharingPermission}
-                participantTypes={resolvedParticipantTypes}
-              />
+              {/* TODO: not show this table if sharedSiteIds and sharedTypes are empty */}
+              {false && (
+                <SharingPermissionsTable
+                  sharedSiteIds={[]}
```

Comment:
For any other reviewers, these two props will be populated in this PR: https://github.com/IABTechLab/uid2-self-serve-portal/pull/163